### ISSUE TITLE
Form Button - Condition

### ISSUE THREAD

**Issue by Torbanio** [munirkamal/gutenberg-forms#145]
Hello,

it would be nice to have a Form button action like "Open Click" and a condition option, where you can activate a group/field or follow an external link. Really miss this function.

And congratulations. Great tool!

Best regards
Torben

**Comment by munirkamal** [comment]
@Torbanio 

First of all, Thank you for your feedback & suggestions. I am going through all those.

Not sure if I get it correctly.

* External Link Button is already possible, you can simply put Gutenberg Button block anywhere in the form.
* Conditionally show field group is also possible but based on other field values.

Can you explain a little more about this feature and what is the possible usecase for it?

**Comment by Torbanio** [comment]
Yes, Gutenberg Button with link is possible but without condition. Sometimes I would like enable/show a group by clicking a button.

**Comment by munirkamal** [comment]
Alright, 

I will explore about this.

So we might have a new action for button like "Trigger Condition" and then from our condition options, we may need to check for the button that has this action and use that button's click event to trigger the condition. 

Marking this as a ToDo now.
